src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -1040,12 +1064,7 @@ export class NotebookModel extends Disposable implements INotebookModel {
 			if (provider && provider !== this._providerId) {
 				this._providerId = provider;
 			} else if (!provider) {
-				this.notebookOptions.executeManagers.forEach(m => {
-					if (m.providerId !== SQL_NOTEBOOK_PROVIDER) {
-						// We don't know which provider it is before that provider is chosen to query its specs. Choosing the "last" one registered.
-						this._providerId = m.providerId;
```

Comment:
I removed this because I noticed when we uninstalled the .NET Interactive extension, I would then have to reconfigure python when reloading the notebook, which can take a non-trivial amount of time. We use SQL as the default kernel everywhere else in this method, so I switched to that instead of Python.